session/pingpong/consumer_balance_tracker.go

```diff
@@ -242,11 +242,13 @@ func (cbt *ConsumerBalanceTracker) handleUnlockEvent(data identity.AppEventIdent
 	err := cbt.recoverGrandTotalPromised(data.ChainID, data.ID)
 	if err != nil {
 		log.Error().Err(err).Msg("Could not recover Grand Total Promised")
+		return
```

Comment:
Even if these result in an error we would want to continue and start `go cbt.lifetimeBCSync(data.ChainID, data.ID)`

Registration status is also handled correctly with the switch. These errors are more for log purpose instead of detecting an actual error.

Reply:
Reverted